app/components/latest-releases-card.ts

```diff
@@ -0,0 +1,54 @@
+import Component from '@glimmer/component';
+import type CourseExtensionIdeaModel from 'codecrafters-frontend/models/course-extension-idea';
+import type CourseIdeaModel from 'codecrafters-frontend/models/course-idea';
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: {
+    courseExtensionIdeas: CourseExtensionIdeaModel[];
+    courseIdeas: CourseIdeaModel[];
+  };
+}
+
+interface Release {
+  announcementUrl?: string;
+  releasedAt: Date;
+  title: string;
+  type: 'challenge' | 'extension';
+}
+
+export default class LatestReleasesCardComponent extends Component<Signature> {
+  get releases(): Release[] {
+    const releases: Release[] = [];
+
+    (this.args.courseExtensionIdeas ?? [])
```

Comment:
```suggestion
    this.args.courseExtensionIdeas
```

The type for this is an array, so don't see why we need to handle false-y values